modular/translations/code/translation_data/ru_names.toml

```
@@ -4234,18 +4279,120 @@ instrumental = "торговым автоматом игрушек Donksoft"
 prepositional = "торговом автомате игрушек Donksoft"
 gender = "male"
 
-["ColMarTech Food Vendor"]
-nominative = "раздатчик еды КолМорТех"
-genitive = "раздатчика еды КолМорТех"
-dative = "раздатчику еды КолМорТех"
-accusative = "раздатчик еды КолМорТех"
-instrumental = "раздатчиком еды КолМорТех"
-prepositional = "раздатчике еды КолМорТех"
-gender = "male"
+["ColMarTech Food Vendor"] # Unique Name
```

```suggestion
["ColMarTech Food Vendor"]
```
Это не уникальные имена, и их следует просклонять.
"Иван передает 5 предметов раздатчик еды «КолМорТех»"

---

поправил